spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -41,11 +37,17 @@ MicroProfile applications that make use of APIs or versions not mentioned below
  - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
  - <<mp-rest-client, MicroProfile Rest Client 2.0>>
 
+A MicroProfile 4.x compatible implementation must pass all the TCKs provided by the MicroProfile individual
+specifications. The Jakarta specifications TCKs are not required.
+
+These APIs and versions guarantee application developers a minimum set of APIs and are not intended to be restrictive.
+MicroProfile applications that make use of APIs or versions not mentioned here are not considered portable.
+
 [[javase]]
-=== Java SE 8
+=== Java SE
 
 Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are unsupported.
-MicroProfile 4.x implementations may provide support for Java SE 9 and beyond, however Java SE 8 compatibility must be maintained.
+MicroProfile 4.x implementations are free to support any Java SE version.
```

Comment:
```suggestion
MicroProfile 4.x implementations are free to provide support for Java SE 8 and beyond.
```

Reply:
Personally, I don't understand why the original statement has to change at all.  It says exactly what we were saying on the call.  Go ahead and use any version of Java SE beyond 8 (inclusive), as long as you Java SE 8 compatibility is maintained.

Reply:
I interpret the part `however Java SE 8 compatibility must be maintained.` as the implementation must support a Java SE 8 runtime. Is this a wrong interpretation?

Also, my understanding from the discussion and consensus in the call was to relax this requirement, and allow implementations to support whatever version they wanted, but without the restriction to always have to support 8.